extensions/healthie/actions/updatePatient/updatePatient.test.ts

```diff
@@ -1,54 +1,144 @@
 import { generateTestPayload } from '@/tests'
-import { getSdk } from '../../lib/sdk/graphql-codegen/generated/sdk'
-import {
-  mockGetSdk,
-  mockGetSdkReturn,
-} from '../../lib/sdk/graphql-codegen/generated/__mocks__/sdk'
-import { updatePatient } from '../updatePatient/updatePatient'
+import { HealthieSdk } from '../../lib/sdk/genql'
+import { updatePatient as actionInterface } from '../updatePatient'
+import { TestHelpers } from '@awell-health/extensions-core'
+import { FieldsValidationSchema } from './config'
 
-jest.mock('../../lib/sdk/graphql-codegen/generated/sdk')
-jest.mock('../../lib/sdk/graphql-codegen/graphqlClient')
+jest.mock('../../lib/sdk/genql', () => ({
+  HealthieSdk: jest.fn().mockImplementation(() => ({
+    client: {
+      mutation: jest.fn().mockResolvedValue({
+        updateClient: {
+          user: {
+            id: 'test-patient',
+          },
+        },
+      }),
+    },
+  })),
+}))
 
-describe('updatePatient action', () => {
-  const onComplete = jest.fn()
+const mockedHealthieSdk = jest.mocked(HealthieSdk)
 
-  beforeAll(() => {
-    ;(getSdk as jest.Mock).mockImplementation(mockGetSdk)
-  })
+describe('Healthie - updatePatient', () => {
+  const {
+    extensionAction: action,
+    onComplete,
+    onError,
+    helpers,
+  } = TestHelpers.fromAction(actionInterface)
 
   beforeEach(() => {
```

Comment:
i see you're using jest.clearAllMocks. totally good way to handle it. FYI, there is also a `clearMocks` function you can accept as part of the return object from `TestHelpers.fromAction()` -- it would only clear onComplete, onError, and helpers.

(in case you need more granular control)